modules/claims_api/lib/bd/bd.rb

```diff
@@ -74,13 +100,14 @@ def upload(claim:, pdf_path:, pctpnt_vet_id: nil, doc_type: 'L122', file_number:
     def generate_upload_body(claim:, doc_type:, pdf_path:, pctpnt_vet_id: nil, file_number: nil, original_filename: nil)
       payload = {}
       auth_headers = claim.auth_headers
-      veteran_name = "#{auth_headers['va_eauth_firstName']}_#{auth_headers['va_eauth_lastName']}"
+      veteran_name = compact_veteran_name(auth_headers['va_eauth_firstName'], auth_headers['va_eauth_lastName'])
       birls_file_num = auth_headers['va_eauth_birlsfilenumber'] || file_number if doc_type != 'L705'
-      claim_id = doc_type == 'L705' ? claim.claim_id : claim.evss_id
+      claim_id = get_claim_id(doc_type, claim)
       file_name = generate_file_name(doc_type:, veteran_name:, claim_id:, original_filename:)
-      participant_id = pctpnt_vet_id if doc_type == 'L705'
+      participant_id = pctpnt_vet_id if %w[L075 L190 L750].include?(doc_type)
```

Comment:
formerly L705 - assuming this is a typo.

Reply:
https://github.com/department-of-veterans-affairs/vets-api/pull/17587/commits/5da8c4d5f92c5136a2c7eb85225fe7db28e5a9fb